docs/source/cmdline.rst

```diff
@@ -13,15 +13,15 @@ Main command
 .. toctree::
    :maxdepth: 1
 
-   generated/man/repronim
+..   generated/man/repronim
```

Comment:
those are supposed to be generated whenever you do 'python setup.py build_manpage'  or 'python setup.py build' (as we do in .travis.yml) ... Michael has RFed it within datalad already to invoke manpage building while doing 'make' under docs/ -- I will adopt it here as well for a more "proper" fix ;)